### ISSUE TITLE
[Improvement] In `HiveTable.java` field `sd` is hidden

### ISSUE THREAD

**Issue by justinmclean** [apache/gravitino#1913]
### What would you like to be improved?

` field `sd` is hidden, this might be a bug?

```java
  private StorageDescriptor buildStorageDescriptor(
      HiveTablePropertiesMetadata tablePropertiesMetadata, List<FieldSchema> partitionFields) {
    StorageDescriptor sd = new StorageDescriptor();
```

### How should we improve?

rename local variable?

**Comment by justinmclean** [comment]
Because the local variable hides a field with the same name.

**Comment by caican00** [comment]
> Because the local variable hides a field with the same name.

What problems will it cause?

**Comment by orenccl** [comment]
This issue seems to be resolved by commit [#4188] Renamed local instance of StorageDescriptor in buildStorageDescriptor (#4276) by Ashwin Kherde on 7/29/24 at 10:01 AM. This issue can now be closed.

**Comment by justinmclean** [comment]
Thanks @orenccl for noticing that this has been fixed.
